config/main.py

```diff
@@ -668,6 +668,10 @@ def _stop_services(config_db):
         if service in services_to_stop:
             services_to_stop.remove(service)
 
+    # First, we stop Monit to prevent any false alarms as other services stop
+    click.echo("Stopping Monit ...")
+    clicommon.run_command("systemctl stop monit", display_cmd=False)
```

Comment:
is pause ok?

Reply:
Please read my findings in the PR dscription.